README.md

```diff
@@ -3,6 +3,27 @@
 
 ![Marathon UI](https://raw.githubusercontent.com/mesosphere/marathon-ui/master/marathon-ui.png "Marathon UI")
 
+## Project Status
+
+The message below was posted on the [marathon-framework google groups|https://groups.google.com/forum/#!msg/marathon-framework/c6qUW8c0_3A/05dh0FjJBgAJ] site:
```

Comment:
Please change the Link to ```[marathon-framework google groups](https://groups.google.com/forum/#!msg/marathon-framework/c6qUW8c0_3A/05dh0FjJBgAJ)``` so that it's properly rendered.